Intersect.Client/Entities/Player.cs

```diff
@@ -114,34 +115,9 @@ public bool IsInParty()
             return Party.Count > 0;
         }
 
-        public bool IsInMyParty(Entity entity)
-        {
-            if (EntityTypes.Player == entity.GetEntityType())
-            {
-                foreach (var member in Party)
-                {
-                    if (member.Id == entity.Id)
-                    {
-                        return true;
-                    }
-                }
-            }
-
-            return false;
-        }
-
-        public bool IsInMyParty(Guid id)
-        {
-            foreach (var member in Party)
-            {
-                if (member.Id == Id)
-                {
-                    return true;
-                }
-            }
+        public bool IsInMyParty(Player player) => Party.Any(member => member.Id == player.Id);
```

Comment:
I honestly would have done `=> IsInMyParty(player.Id)` just to limit how many methods are doing essentially the same thing, or have done `Party.Contains(player)` here (first is probably better here).

Reply:
Good point, overlooked that.